### ISSUE TITLE
Attribute value of true is not rendering correctly in XMLBuilder

### ISSUE THREAD

**Issue by ATXadam** [NaturalIntelligence/fast-xml-parser#611]
* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [x] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Description
When using XMLBuilder to render an attribute with a value (string or bool) of 'true', XMLBuilder does not render the value as a string in the attribute.

### Input
```json
{"test": {"@_isTrue": true}}
```
### Code
```js
const { XMLBuilder } = require('fast-xml-parser')
let builder = new XMLBuilder({ ignoreAttributes: false});
builder.build({"test": {"@_isTrue": true}})
```

### Output
```xml
<test isTrue></test>
```

### expected data
```xml
<test isTrue="true"></test>
```

**Would you like to work on this issue?**
- [x] Yes
- [ ] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates. Visit [SoloThought](https://solothought.com) to know about recent features.

**Comment by ATXadam** [comment]
After starting to look through the code, I saw suppressBooleanAttributes in the XMLBuilder options, setting this to false resolves the issue. https://naturalintelligence.github.io/fast-xml-parser/ "Allow Boolean Attributes" did not set this flag going from JSON to XML it appears.

```js
const { XMLBuilder } = require('fast-xml-parser');
let builder = new XMLBuilder({ ignoreAttributes: false, suppressBooleanAttributes: false});
builder.build({"test": {"@_isTrue": true}});
```

Yields
```xml
<test isTrue="true"></test>
```
